### ISSUE TITLE
Tile window to south/north

### ISSUE THREAD

**Issue by stefanct** [mate-desktop/marco#602]
I would like to be able to tile not only vertically east/west but also horizontally north/south, i.e. I would like to have a shortcut similar to "tile window to west (left) side of screen" but for north (top) and south (bottom) to make it snap to the top/bottom border of the current screen and resize it to take the whole width but only half the height.
I am using marco 1.20 on Debian Buster but AFAICT there is no sign of such functionality in current release (1.24.0).

**Comment by stefanct** [comment]
No, not ignored - actively and decisively avoided because it does not make any sense for feature requests at all.

**Comment by RichardNeill** [comment]
Please can I beg to differ. This may appear spurious on a regular widescreen (one might think: who would, after all want to tile a window that long and thin?), but it is exactly what I was wanting now that I have a 2nd screen in portrait mode, where north + south is the obvious way to tile 2 windows, which become 9:8 ratio.  Thanks for your consideration.

[What I use for coding is a big 4k central screen (16:9 landscape), flanked by a pair of smaller 2k screens (16:9 portrait). This allows 8 tiled windows, but it would really help if there was a quick way to tile a pair of windows on the side monitors, to top + bottom.]

**Comment by RichardNeill** [comment]
See also Ubuntu bug: https://bugs.launchpad.net/ubuntu-mate/+bug/1905141

**Comment by allanlaal** [comment]
use case: 

I sometimes set 1..N of my 3..N displays in vertical mode


use case:

in some locations I use old 4:3 screens

**Comment by allanlaal** [comment]
workaround: I use [quicktile](https://github.com/ssokolow/quicktile)

**Comment by Lantizia** [comment]
Has this been fixed now?

https://github.com/mate-desktop/marco/blob/master/src/50-marco-window-key.xml.in

I now see...

```
	<KeyListEntry
	name="move-to-side-n"
	description="Move window to north (top) side of screen" />

	<KeyListEntry
	name="move-to-side-s"
	description="Move window to south (bottom) side of screen" />```

**Comment by RichardNeill** [comment]
Sadly, that's not the fix I was hoping for. That shortcut, move-to-side-s, just moves the window to the bottom of the screen (without re-sizing it). What I was hoping for was TILE-to-south i.e. "The window is moved and resized so as to occupy the bottom half of the monitor".

**Comment by Lantizia** [comment]
Ah yes!, good point... I've also been tripped up by that... should have realised.

This does seem like an embarrassing thing to be against @stefanct especially in light of so many people with rotatable 16:9 monitors that they use in portrait mode for work!
